### PR TITLE
fix(HMS-1242): atomically read statistics

### PR DESCRIPTION
**Pull request by lzap**
I noticed the read atomic overlook, a variable that must be threadsafe must be read and written via `atomic` function. This could possibly lead to stale value read.

Additionally, I noticed that the Redis client creates connection pool of CPUs * 10. We know for fact how big the pool should be - it needs to be numPollers + 1 + some room for statistics calls (usually just 1 but to give it some room 10 should be a good limit).

I renamed some variables to better reflect their purpose.

### CODE REVIEW

File: pkg/worker/redis.go [hunk 1]
```diff
@@ -219,6 +220,6 @@ func (w *RedisWorker) Stats(ctx context.Context) (Stats, error) {
 
 	return Stats{
 		EnqueuedJobs: uint64(count),
-		InFlight:     w.inFlight,
+		InFlight:     atomic.LoadInt64(&w.inFlight),
```
Comment (lzap) [inline pkg/worker/redis.go:223]: This is the bugfix, the rest is just renaming of variables and setting up PoolSize to more reasonable number.

Comment (ezr-ondrej) [reply pkg/worker/redis.go:223]: I thought you actually need to define it as atomic... ILT

**Comment by ezr-ondrej** [comment]
/retest
but the bot needs to learn as well

**Comment by ezr-ondrej** [comment]
/retest
The IQE pod failed to start now :thinking:

**Review by ezr-ondrej — APPROVED** [review]

**Pull request merged**